script.js

```diff
@@ -0,0 +1,32 @@
+
+function addTask() {
+  const description = addInput.value;
+  if (description) {
+    Task.addTask(description);
+    addInput.value = '';
+    renderTasks();
+  }
+}
+
+renderTasks();
+
+addButton.addEventListener('click', () => {
+  addTask();
+});
+
+addInput.addEventListener('keypress', (event) => {
+  if (event.key === 'Enter') {
+    addTask();
+  }
+});
+
+clearAllButton.addEventListener('click', () => {
+  const tasks = Task.getTasks();
+  const newTasks = tasks.filter((task) => !task.completed);
+  newTasks.forEach((task, index) => {
+    task.id = index + 1;
+  });
+
```

Comment:
**Event Listeners**: The event listeners are correctly set up for the buttons and keypress events. However, it is good practice to remove event listeners when they are no longer needed to avoid potential memory leaks.